src/index.html

```diff
@@ -6,9 +6,82 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="#">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo__link">
+        <a class="logo__img">
+          <img src="./images/logo.png" alt="logo">
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__conteiner">
+          <li class="nav__item">
+            <a
+            href="#"
+            class="nav__link is-active">
+            Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+            href="#"
+            class="nav__link">
+            Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+            href="#"
+            class="nav__link">
+            Smartphones
+            </a>
+          </li>
```

Comment:
<img width="1122" alt="Spaces between tags" src="https://github.com/mate-academy/layout_moyo-header/assets/86789161/7031a6e8-e99e-4018-8c9a-6506923d0227">

Reply:
The same in other places